src/main/java/org/mvel2/optimizers/impl/refl/ReflectiveAccessorOptimizer.java

```diff
@@ -451,7 +451,8 @@ private Accessor compileGetChain() {
       throw e;
     }
     catch (NullPointerException e) {
-      throw new PropertyAccessException("null pointer: " + new String(expr, start, length), this.expr, this.st, e, pCtx);
+      LOG.log(Level.WARNING, String.format("null pointer: %s", new String(expr, start, length)));
```

Comment:
You're only modifying ReflectiveAccessorOptimizer so you might be introducing a new discrepancy with asm mode (ASMAccessorOptimizer, usually kicks-in after a configurable number of executions) and plain interpreted mode (when you call `MVEL.eval(expression, paramMap)`)